patch/kconfig-inclusions

```diff
@@ -126,6 +126,182 @@ CONFIG_SENSORS_EMC2305=m
 [marvell-arm64]
 CONFIG_MARVELL_AC5_WATCHDOG=m
 
+[pensando]
+CONFIG_ARCH_PENSANDO=y
+CONFIG_ARCH_PENSANDO_ELBA_SOC=y
+CONFIG_PENSANDO_SOC_CAPMEM_HUGEPAGE=y
+CONFIG_PENSANDO_SOC_PENFW=y
+CONFIG_EDAC_ELBA=y
+CONFIG_IRQ_PENSANDO=y
+CONFIG_MFD_PENSANDO_ELBASR=y
+CONFIG_MMC_SDHCI_IO_ACCESSORS=y
+CONFIG_RESET_ELBASR=y
+CONFIG_UIO=y
+CONFIG_UIO_PCIEMAC=y
+CONFIG_UIO_LINKMAC=y
+CONFIG_UIO_PENMSI1=y
+CONFIG_UIO_PENGIC=y
+CONFIG_UIO_PENMSI=y
+CONFIG_ARM64_VA_BITS_39=y
```

Comment:
Any specific reason 39-bit virtual addresses are being used instead of 48-bit virtual addresses? From a quick glance, arm64 CPUs support both 4K pages (48-bit virtual addresses) and 64K pages (39-bit virtual addresses), and I think it would be preferred to keep the 4K page size (if nothing else, then to match what marvell-arm64 has).

Reply:
> Any specific reason 39-bit virtual addresses are being used instead of 48-bit virtual addresses? From a quick glance, arm64 CPUs support both 4K pages (48-bit virtual addresses) and 64K pages (39-bit virtual addresses), and I think it would be preferred to keep the 4K page size (if nothing else, then to match what marvell-arm64 has).

This is due to NOC (Network-on-Chip->NOC, which provides NOC proc interface to CPU) in ELBA having only 40 bits.

Reply:
Ah, meaning the NOC can only support virtual addresses up to 40 bits?

Reply:
Yes. NOC in ELBA is 40 bits..